src/routes/AppContent/index.tsx

```diff
@@ -54,7 +54,9 @@ const AppContent = () => {
         if (activeAccountData) {
             subscribeP2PSettings({});
         }
-
+        return () => {
+            setP2PSettings({});
+        };
```

Comment:
because the p2p_settings data is never cleared in the local_storage when the user either reloads the page or logs out then logs back in, the currency dropdown will always fetch the data from p2p_settings before it gets updated. So we need to clear it first, so it doesn't pass this if check in BuySellTable where it sets the filtered currency

<img width="678" alt="Screenshot 2024-08-06 at 10 45 00 AM" src="https://github.com/user-attachments/assets/d5ab1f13-7eed-49f0-b8f1-38119f3c80fa">